nav/filter_lidar_data/src/ignore_lidar_node.cpp

```diff
@@ -32,10 +32,10 @@ IgnoreLidarNode::IgnoreLidarNode(ros::NodeHandle nh)
     
     // Set rectangular bounds where lidar data needs to be adjusted. Based on the 2nd and 3rd waypoint
     // TODO: Tune padding  
-    bounds_[0].x = second_pose_stamped.pose.position.x + 1.5;
-    bounds_[0].y = second_pose_stamped.pose.position.y + 4.0;
-    bounds_[1].x = third_pose_stamped.pose.position.x - 1.0;
-    bounds_[1].y = third_pose_stamped.pose.position.y - 4.0;
+    bounds_[0].x = second_pose_stamped.pose.position.x + 3.0;
```

Comment:
Is this change intentional?

Reply:
to increase the bounds (max GPS offset we estimate is like, 2m, so we did 3m in all 4 cardinal directions). I think this change shouldn't leave an impact since ivy is changing it anyways, and I changed local/global costmaps to subscribe from /scan rather than /scan_modified